RichardSzalay.MockHttp.Tests/MockHttpMessageHandlerTests.cs

```diff
@@ -157,26 +157,6 @@ public void Should_return_fallback_for_unmatched_requests()
             Assert.Equal("Awesome", result.ReasonPhrase);
         }
 
-        [Fact]
-        public void Should_return_fallbackresponse_for_unmatched_requests()
```

Comment:
Why did you delete these tests?

Reply:
Because they are unit testing the FallbackResponse property which we just deleted.

Reply:
Also, the behavior that was being unit tested in `Should_return_fallbackresponse_for_unmatched_requests` against the `FallbackResponse` property is unit tested in `Should_return_fallback_for_unmatched_requests` against the `Fallback` property and the `Should_not_match_expect_out_of_order`  unit test relied on `FallbackResponse` having a property named `StatusCode`. Fallback does not have have such a property so I couldn't figure out a obvious replacement for the unit test.

Reply:
Gotcha. I agree with the first, but `Should_not_match_expect_out_of_order` is a critical piece of behavior so I'll take your branch and reimplement it before merging.